filters/diacritic_filter/README.md

```diff
@@ -0,0 +1,23 @@
+## Diacritics filter
+
+## What type of a filter is this?
+
+This filter checks whether any character in the sentence has a diacritic.
+
+From Merriam-Webster: Diacritics are marks placed above or below (or sometimes next to) a letter in a word to indicate a particular pronunciation -- in regard to accent, tone, or stress — 
```

Comment:
The format of the reference should be : 
According to Merriam-Webster [[1]](link), didactics are the marks ...